.github/workflows/ruby.yml

```diff
@@ -34,6 +31,13 @@ jobs:
       - name: Install PostgreSQL 11 client
         run: |
           sudo apt-get -yqq install libpq-dev
+      - name: Cache gems
+        uses: actions/cache@preview
```

Comment:
@StefanGajic why are you using the `preview` version of this action? Do you use some new feature it adds?

Reply:
I looked at the docs, and I should be using the latest version which is `uses: actions/cache@v2`